fastlane/Helpers/Match.swift

```diff
@@ -33,6 +33,51 @@ enum Match {
                 force: .userDefined(isForce)
             )
         }
+        updateCodeSigning(type: type, appIdentifier: appIdentifier)
+    }
+    
+    static func updateCodeSigning(type: MatchType, appIdentifier: [String]) {
+        // Update Code signing from automatic to manual
+        switch type {
+        case .development:
+            if appIdentifier.first == Constant.productionBundleId {
+                updateCodeSigningSettings(
+                    path: Constant.projectPath,
+                    useAutomaticSigning: .userDefined(false),
+                    targets: .userDefined([Constant.projectName]),
+                    buildConfigurations: .userDefined(["Debug Production"]),
```

Comment:
I think we can define buildConfiguration as enum var
```
        var buildConfiguration: String {
             switch self {
             case .development: return "Debug"
             case .adHoc, .appStore: return "Release"
             }
         } 
``` 
and here we change 
```suggestion
                    buildConfigurations: .userDefined(["\(type.buildConfiguration) Production"]),
```